project_issue_code/views/project_issue_view.xml

```diff
@@ -40,7 +40,14 @@
         <field name="inherit_id" ref="project_issue.view_project_issue_filter" />
         <field name="arch" type="xml">
             <xpath expr="//field[@name='name']" position="attributes">
-                <attribute name="filter_domain">['|', ('name', 'ilike', self), ('issue_code', 'ilike', self)]</attribute>
+                <attribute name="filter_domain">['|', '|', '|', '|', '|', \
+                    ('partner_id','child_of',self), \
+                    ('description','ilike',self), \
+                    ('email_from','ilike',self), \
+                    ('name','ilike',self), \
+                    ('name', 'ilike', self), \
```

Comment:
This kind of notation is always a bit confusing, but it seems to me that all the conditions are put together with `OR`, so why is it needed to repeat this one twice?